x-pack/plugins/reporting/public/components/report_info_button.tsx

```diff
@@ -0,0 +1,141 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { EuiButtonIcon, EuiCallOut, EuiPopover } from '@elastic/eui';
+import { get } from 'lodash';
+import React, { Component, Fragment } from 'react';
+import { JobInfo, jobQueueClient } from '../lib/job_queue_client';
+
+interface Props {
+  jobId: string;
+}
+
+interface State {
+  isLoading: boolean;
+  isPopoverOpen: boolean;
+  calloutTitle: string;
+  info: JobInfo | null;
+}
+
+export class ReportInfoButton extends Component<Props, State> {
+  private mounted?: boolean;
+
+  constructor(props: Props) {
+    super(props);
+
+    this.state = {
+      isLoading: false,
+      isPopoverOpen: false,
+      calloutTitle: 'Job info',
+      info: null,
+    };
+  }
+
+  public renderInfo() {
+    const { info } = this.state;
+    if (!info) {
+      return null;
+    }
+
+    const iGet = (path: string) => get(info, path, 'n/a');
+
+    // TODO browser type
```

Comment:
How hard is this to plumb up? I can take that in a sub-task :)

Reply:
I think it would be worth the effort! Right now, browser type is not in [the schema of data that gets indexed to manage the reporting jobs](https://github.com/elastic/kibana/blob/master/x-pack/plugins/reporting/server/lib/esqueue/helpers/create_index.js#L9). It would need to be added to the fields of data and populated when jobs are created.

Reply:
Coolio, don't want to block this from merging, so I'll do my own PR once this is in master

Reply:
I put another todo of "queue method" in. It's another thing we don't have in the record today: whether the job was queued by clicking the button in the UI, or using the POST URL. It would be nice to have, as (I think) the code path between the options is a little different.